include/pisa/query/algorithm/and_query.hpp

```diff
@@ -28,41 +24,30 @@ struct and_query {
             ordered_cursors.push_back(&en);
         }
 
-
         // sort by increasing frequency
         std::sort(ordered_cursors.begin(), ordered_cursors.end(), [](Cursor *lhs, Cursor *rhs) {
-            return lhs->docs_enum.size() < rhs->docs_enum.size();
+            return lhs->size() < rhs->size();
```

Comment:
How come this works, cursors don't seem to have `size`, `docid` and so on, do they? What am I missing?

Reply:
```
using cursor          = typename Index::document_enumerator;
```